app/assets/stylesheets/application.scss

```diff
@@ -61,3 +61,17 @@ body {
   text-align: center;
 
 }
+
+#map-container {
+   height: 400px;
+   border-radius: 16px 16px;
+   border-color: #fff;
+   border-style: solid;
+   box-shadow: 2px 2px 10px #B1B1B1;
```

Comment:
Color literals like `#B1B1B1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B1B1B1` should be written as `#b1b1b1`<br>Line should be indented 2 spaces, but was indented 3 spaces